.appveyor.yml

```diff
@@ -99,6 +69,5 @@ build_script:
   - md build
   - cd build
   - cmake --version
-  - if NOT DEFINED PLATFORM cmake %AIRPCAP% %REMOTE% -DPacket_ROOT=c:\projects\libpcap\Win32\%SDK% -G"%GENERATOR%" -DPacket_ROOT=c:\projects\libpcap\Win32\%SDK% -DPacket_ROOT=c:\projects\libpcap\Win32\%SDK% -G"%GENERATOR%" -DAirPcap_ROOT=c:\projects\libpcap\Win32\Airpcap_Devpack -G"%GENERATOR%" ..
-  - if DEFINED PLATFORM cmake %AIRPCAP% %REMOTE% -DPacket_ROOT=c:\projects\libpcap\Win32\%SDK% -G"%GENERATOR%" -DPacket_ROOT=c:\projects\libpcap\Win32\%SDK% -DPacket_ROOT=c:\projects\libpcap\Win32\%SDK% -G"%GENERATOR%" -DAirPcap_ROOT=c:\projects\libpcap\Win32\Airpcap_Devpack -G"%GENERATOR%" -A %PLATFORM% ..
```

Comment:
This one shouldn't be removed, either.

This is really an if/else case, but the AppVeyor YAML doesn't support that; it does two different commands so that it can include `-A` for non-default platforms (i.e., other than IA-32) and not include it for the default platform (IA-32).

Defining `PLATFORM` as either an empty string or as "-A <whatever>" might allow that to be done in one command, but that's unrelated to dropping AirPcap support, and should probably be done in a separate change, so put back the `if DEFINED PLATFORM`... line, removing ` %AIRPCAP` from it as well (complete with removing the blank, as well as removing it from the `if NOT DEFINED PLATFORM`line, so there aren't two spaces in a row.

Reply:
The AppVeyor build is fixed.